lib/extractor/docker-archive/layer.ts

```diff
@@ -83,6 +83,10 @@ function getLayersContentAndArchiveManifest(
     .map((layerName) => layers[layerName])
     .reverse();
 
+  if (filteredLayers.length === 0) {
+    throw new Error("We found no layers in the provided image");
+  }
```

Comment:
Do we have a test case for this, any way we can add coverage?

Reply:
Yeees, was just thinking that! 👍

Reply:
Done!